app/index/controller.js

```diff
@@ -87,15 +87,19 @@ export default Controller.extend({
   },
 
   /**
-   * All activities filtered by the selected day
+   * All activities filtered by the selected day and the current user
    *
    * @property {Activity[]} _activities
    * @private
    */
-  @computed('date', '_allActivities.@each.{start,isDeleted}')
-  _activities(day, activities) {
+  @computed('date', '_allActivities.@each.{start,user,isDeleted}', 'user')
+  _activities(day, activities, user) {
     let activitiesThen = activities.filter(a => {
-      return a.get('start').isSame(day, 'day') && !a.get('isDeleted')
+      return (
+        a.get('start').isSame(day, 'day') &&
+        a.get('user.id') === user.get('id') &&
+        !a.get('isDeleted')
```

Comment:
For my taste, the filter is duplicated so much that it might be forth factoring it out. For lack of a better name, how about
`activities.filter(currentPersonal)`

Reply:
I personally don't like extracting a filter which filters different entities..